charts/opensearch-dashboards/README.md

```diff
@@ -92,9 +92,9 @@ helm uninstall my-release
 | `opensearchDashboardsYml.defaultMode` | Allow you to set the defaultMode for the opensearch_dashboards.yml mounted as configMap | |
 | `dashboardAnnotations` | Allows you to configure custom annotation in the deployement of the OpenSearchDashboards container | {} |
 | `serviceMonitor.enabled` | Enables the creation of a [ServiceMonitor] resource for Prometheus monitoring. Requires the Prometheus Operator to be installed in your Kubernetes cluster. | `false` |
+| `serviceMonitor.portName` | Name of the port in the OpenSearch Dashboards service that exposes metrics. This should match the port name defined in your OpenSearch service configuration. Applicable only if `serviceMonitor.enabled` is set to `true`. | `metrics` |
```

Comment:
Does this mean the name of `serviceMonitor.portName` should be same as OpenSearch service created by the chart ? If so the value passed is `portName: metrics`, so is `metrics` is the OpenSearch service name?

Reply:
Yes. Correct. `metrics` is the name of the OpenSearch service which exposes the metrics. 
Reference from the existing `Service` template: https://github.com/opensearch-project/helm-charts/blob/4253842c1e4d3ac6d4aee294e905c1f20469adc2/charts/opensearch/templates/service.yaml#L66

Reply:
Got it thanks @VILJkid then should we use the same syntax ?
`{{ .Values.service.metricsPortName | default "metrics" }}` 
@peterzhuamazon @TheAlgo @getsaurabh02

Reply:
Sounds good! Then creating `portName` won't be necessary. We can directly use the syntax to fetch the service name.

Thanks @prudhvigodithi! I'll make the necessary changes.

Reply:
Thanks @prudhvigodithi,

The changes are implemented in the latest commits. Kindly review and let me know if it needs any further changes.